docs/Developers/Newcomers_Guide.rst

```diff
@@ -141,6 +141,15 @@ Now you can start working on it.
 
 .. note::
 
+    As stated before, you should never work on an issue without any
+    assignment. Fortunately, cobot is here to help you! So, if you are
+    interested in picking up an issue just write in the gitter chat the
+    following command:
+
```

Comment:
Let it look like a command. It'll look better this way -  
::

    cobot assign <issue_link>